content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Generate Istio Metrics Without Mixer
+description: In-proxy generation of service-level metrics.
+weight: 70
+---
+
+{{< warning >}}
+
+This document describes **Alpha** quality features. They have been exposed *only* to enable further testing
+and evaluation across the community.
+
+The functionality described here is NOT considered stable. No performance or stability guarantees are provided
+at this time.
+
+{{< /warning>}}
+
+Istio 1.3 adds experimental support to generate service-level HTTP metrics
+directly in the Envoy proxies. This feature lets you continue to monitor your service meshes using the tools Istio provides
+without needing Mixer. This support is the first step towards the new
+extensibility architecture for Istio. This architecture puts Mixer-like functionality directly in the Envoy proxies.
+
+The in-proxy generation of service-level metrics replaces the following HTTP metrics that Mixer currently
+generates:
```

Comment:
worth adding that in-proxy does not currently support other metrics like latency?